systest/scripts/install_lbaasv2.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -ex
+
+GLOBAL_ROUTED_MODE=False
+OS_CONTROLLER_IP=`/tools/bin/tlc --sid ${TEST_SESSION} symbols \
+    | grep openstack_controller1ip_data_direct \
+    | awk '{print $3}' | xargs`
+SSH_CMD="ssh -i /home/jenkins/.ssh/id_rsa_testlab -o StrictHostKeyChecking=no testlab@${OS_CONTROLLER_IP}"
+BIGIP_IP=`${SSH_CMD} "cat /home/testlab/ve_mgmt_ip"`
+BIGIP_IP=${BIGIP_IP%%[[:cntrl:]]}
+NETWORK_TYPES=${TEST_TENANT_NETWORK_TYPE}
+AGENT_LOC=git+https://github.com/F5Networks/f5-openstack-agent.git@${TEST_OPENSTACK_DISTRO}
+DRIVER_LOC=git+https://github.com/F5Networks/f5-openstack-lbaasv2-driver.git@${TEST_OPENSTACK_DISTRO}
+NEUTRON_DRIVER_LOC=https://raw.githubusercontent.com/F5Networks/neutron-lbaas/stable/${TEST_OPENSTACK_DISTRO}/neutron_lbaas/drivers/f5/driver_v2.py
+
+# Since we don't do anything special in the __init__.py file, we can pull it from anywhere for now
+NEUTRON_INIT_LOC=https://raw.githubusercontent.com/F5Networks/neutron-lbaas/v9.1.0/neutron_lbaas/drivers/f5/__init__.py
+
+
+EXTRA_VARS="agent_pkg_location=${AGENT_LOC} driver_pkg_location=${DRIVER_LOC} neutron_lbaas_driver_location=${NEUTRON_DRIVER_LOC}"
+EXTRA_VARS="${EXTRA_VARS} neutron_lbaas_init_location=${NEUTRON_INIT_LOC} restart_all_neutron_services=true remote_user=testlab"
+EXTRA_VARS="${EXTRA_VARS} f5_global_routed_mode=${GLOBAL_ROUTED_MODE} bigip_netloc=${BIGIP_IP} agent_service_name=f5-openstack-agent.service"
+EXTRA_VARS="${EXTRA_VARS} use_barbican_cert_manager=True neutron_lbaas_shim_install_dest=/usr/lib/python2.7/site-packages/neutron_lbaas/drivers/f5"
+
+if [[ $NETWORK_TYPES == 'vlan' ]]; then
+    ADVERTISED_TUNNEL_TYPES=""
+else
+    ADVERTISED_TUNNEL_TYPES=${NETWORK_TYPES}
+fi
+EXTRA_VARS="${EXTRA_VARS} advertised_tunnel_types=${ADVERTISED_TUNNEL_TYPES}"
+
+echo [hosts] > ansible_conf.ini
+echo "${OS_CONTROLLER_IP} ansible_ssh_common_args='-o StrictHostKeyChecking=no' host_key_checking=False ansible_connection=ssh ansible_ssh_user=testlab ansible_ssh_private_key_file=/home/jenkins/f5-openstack-lbaasv2-driver/id_rsa_testlab" >> ansible_conf.ini
+
+git clone -b ${TEST_OPENSTACK_DISTRO} https://github.com/zancas/f5-openstack-ansible.git
```

Comment:
Also, the TEST_OPENSTACK_DISTRO var is not appropriate for all branches in the driver, as above.